add game description, motivation and gif (#8)
VaniOFX/DQN-divergence: blog_post.md

We run each experiment with 25 different random seeds to achieve more statistically sound results, while taking into account our computational budget. If the maximal absolute Q-value predicted in any of the last 20 episodes is above the threshold $$\frac{1}{1-\gamma}$$, we say soft divergence occurs. 
<!--- At the end, we compare the configurations by counting how many times each of them has diverged. -->

> Even though the original paper uses a convolutional neural network to play Atari games, we limit ourselves to simpler environments so that we can use a fully-connected neural network with a single layer of size 128. We pick the classical control theory environments made available by [OpenAI gym](https://gym.openai.com/envs/#classic_control).

Reviewer: ```suggestion
Even though the original paper uses a convolutional neural network to play Atari games, we limit ourselves to simpler environments given computation and time constraints. We use a **fully-connected** neural network with **a single hidden layer (excluding input and output layers) of size 128.**, mapping from input states to a discrete set of actions. We use **ReLU** activation functions at each layer before the output layer. We consider classical control theory environments made available by [OpenAI gym](https://gym.openai.com/envs/#classic_control).
```

### [Cart Pole](https://gym.openai.com/envs/CartPole-v1/)
![cartpole](img/cartpole.gif)

> The agent tries to balance a pole on a cart by applying a rightward or a leftward force. For every time step the pole remains upright (less than 15 degrees from vertical), the agent recieves a reward of +1. This problem is very easy to solve. Therefore, we chose this as a representative of problems with low-level difficulty.

Reviewer: ```suggestion
In the Cart Pole environment, the agent tries to balance a pole on a cart by applying a rightward or a leftward force. For every time step the pole remains upright (less than 15 degrees from vertical), the agent receives a reward of +1. Since his problem is considered relatively easy to solve. we chose it as a representative of problems with low difficulty.
```

Author: remove "to the cart"

### [Acrobot](https://gym.openai.com/envs/Acrobot-v1/)
![acrobot](img/acrobot.gif)

> The agent tries to swing up a two-link robot arm above the base by applying a clockwise or anti-clockwise torque. This problem is more difficult than the previous one. We select this as a representative of problems with mid-level difficulty.

Reviewer: ```suggestion
In the Acrobot environment, the agent tries to swing up a two-link robot arm above the base by applying a clockwise or anti-clockwise torque. This problem is considered more difficult than the previous one, so we select it as a representative of problems with mid-level difficulty.
```

Author: remove "force" after "torque".
add "so we select it"

### [Mountain Car](https://gym.openai.com/envs/MountainCar-v0/)
![mountaincar](img/mountaincar.gif)

> The agent starts a car at the bottom of a valley and tries to drive it up the right hill. However, the car's engine is not strong enough to do so in a single pass. Instead it has to go back and forth between the left and right hill to build momentum. This problem is quite challenging. We pick this as a representative of problems with high-level difficulty.

Reviewer: ```suggestion
In the Mountain Car environment, the agent starts a car at the bottom of a valley and tries to drive it up the right hill. However, the car's engine is not strong enough to do so in a single pass. Instead it has to go back and forth between the left and right hill to build momentum. This problem is quite challenging, so we choose it as a representative of problems with high-level difficulty.
```
